docx/header.py

```diff
@@ -8,12 +8,11 @@
     absolute_import, division, print_function, unicode_literals
```

Comment:
@scanny ok here's my first stab at a unit test. I modeled it after this commit:

https://github.com/python-openxml/python-docx/commit/bddd27a895ee8d3b1f83b860f96255d434515421

The first acceptance test passes now, as does this unittest!